src/components/Tooltip/__tests__/Tooltip.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Tooltip } from '../Tooltip';
+
+describe('<Tooltip> that renders a Tooltip', () => {
+    let view: RenderResult;
+
+    it('should render default Tooltip correctly', () => {
+        view = render(
+            <Tooltip placement="bottom" content="content">
+                <>Hover me</>
+            </Tooltip>
+        );
+
+        expect(view.container).toMatchSnapshot();
+        expect(view.container).toBeInTheDocument();
+    });
+
+    it('should render Tooltip in disabled mode if content is empty', () => {
+        view = render(
+            <Tooltip content="foo">
+                <>Hover me</>
+            </Tooltip>
+        );
+
+        expect(view.container).toBeInTheDocument();
```

Comment:
In original test checked prop disabled, but here we can't do it, should we leave as it is?

Reply:
We need an actual DOM element to check on. If you find what element it is (see comment above), you can test `.toBeDisabled` against that one. 
You can also check the user behaviour. I'm not 100% sure how a disabled tooltip looks like, but you I guess it is just doesn't show. So if you get a sequence of user events above to make the tooltip appear, you can repeat those here and expect that the tooltip won't be there.

Reply:
It returns the same 
![image](https://github.com/textkernel/oneui/assets/104063210/6bd4cfdc-462e-4f16-ad98-3bd3ee003d74)

Reply:
When call expect(view.container).toBeDisabled(); it returns error: 
![image](https://github.com/textkernel/oneui/assets/104063210/7e0a7bc2-5ad9-4ba2-9f16-ba3729aa3c61)